### PR TITLE
Roughly half complete SCP-079

### PR DESCRIPTION
**Pull request by PundaVR**
Always open for feedback

### CODE REVIEW

File: classes/scp/scp079.md [hunk 1]
```diff
@@ -6,19 +6,27 @@
...
 
-| Abilities/Power
+| SCP-079 has a wide range of abilities, from controling doors to activating tesla gates. The full list of it's abilities goes as follows:
+- Opening/Closing Doors
+- Locking Doors
+- Blackout (Locks all doors and turns the lights off in a room, this is excellent if SCP-173 has breached containment) as SCP-173 can kill everyone in that room instantly unless they have a flashlight)
```
Comment (Llama-Karl) [inline classes/scp/scp079.md:22]: or if a player has a night vision scope attached

Comment (PundaVR) [reply classes/scp/scp079.md:22]: Thank you I will add that.

**Comment by Lunch9992** [comment]
Looking good. I happened to notice some typos, not the best choice of wording and some paragraph structure problems but they should be fine as I always rewrite anything put onto the site anyway. I'll have a look at it tomorrow to see if anything conflicts with changes in the Parabellum update as well

**Comment by PundaVR** [comment]
> Looking good. I happened to notice some typos, not the best choice of wording and some paragraph structure problems but they should be fine as I always rewrite anything put onto the site anyway. I'll have a look at it tomorrow to see if anything conflicts with changes in the Parabellum update as well

I closed this because as as you said there were a few typos and things that could be improved on, going to write up entrance and surface as well as the other parts that need to be completed before submitting another request.

**Comment by Lunch9992** [comment]
You can leave it open for however long you want if you need to. Site and video progress at the moment is well beyond us needing specific sections of the guide since we still have quite a bit to do.

**Comment by PundaVR** [comment]
Completed SCP-079, feel free to review and give feedback.

**Pull request closed without merging**